includes/wpml/class-sensei-wpml.php

```diff
@@ -34,4 +57,73 @@ public function sensei_after_sending_email() {
 		*/
 		do_action( 'wpml_restore_language_from_email' );
 	}
+
+	/**
+	 * Set language details for the lesson when it is created.
+	 *
```

Comment:
Should this be [`@internal`](https://senseip2.wordpress.com/2022/10/19/marking-code-as-internal/)?